### PR TITLE
chore: prepare release 9.0.0-alpha.7

### PR DESCRIPTION
**Pull request by kulmann**
Prepare pre-release `v9.0.0-alpha.7`.

### CODE REVIEW

File: pnpm-lock.yaml
Comment (JammingBen) [inline pnpm-lock.yaml:1]: Why is this included? 🤔

Comment (kulmann) [reply pnpm-lock.yaml:1]: no idea 🤔

**Review by JammingBen — APPROVED** [review]
